dorpsgek_github/aiohttp_apps/github.py

```diff
@@ -0,0 +1,23 @@
+import http
+import logging
+
+from aiohttp import web
+from dorpsgek_github.processes import github
+
+log = logging.getLogger(__name__)
+
+
+async def github_handler(request):
+    headers = request.headers
+    data = await request.read()
+
+    try:
+        await github.process_request(headers, data)
+        return web.Response(status=http.HTTPStatus.OK)
+    except Exception:
```

Comment:
catching Exceptions is generally considered Bad Form, because it catches everything, including keyboard interrupts

Reply:
Generally, yes. In this case, there is nothing else. I want that on any error, the exception is logged and the user is served an internal server error. Funny enough, if I wouldn't do it here, aiohttp would do it himself a bit later. As you don't want your application to crash if 1 request fails to be processed. So there is not much else.

Reply:
Also, I found out, that KeyboardInterrupt is handled by asyncio, and always work. This because these entries are in coroutines, and not in the main thread/routine :D